src/App.tsx

```
@@ -50,9 +65,10 @@ function App() {
     fetchWeatherData(location);
   }, []);
 
-  const handleSearch = (e: React.FormEvent) => {
-    e.preventDefault();
-    fetchWeatherData(location);
+  const handleSearch = (searchLocation: string) => {
```

ESLint

[error] 'handleSearch' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)